### PR TITLE
Conclude `MCNC` tournament articles

### PR DESCRIPTION
**Pull request by xz1z1z1z1z**

### CODE REVIEW

File: wiki/Tournaments/MCNC/2021/en.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
   - MCNC2021
+  - MCNC 2021
 ---
 
 # osu!mania 4K Chinese National Cup 2021
 
 ![MCNC 2021 logo](img/logo.jpg)
 
-The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) is a country-based osu!mania tournament hosted by the \[Crz\]Team. It is the third instalment of the osu!mania 4K Chinese National Cup.
+The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the third installment of the osu!mania 4K Chinese National Cup.
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:12]: british spelling

```suggestion
The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the third instalment of the osu!mania 4K Chinese National Cup.
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 2]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
 
 - [Discussion thread (English)](https://osu.ppy.sh/community/forums/topics/1260186)
 - [Discussion thread (Chinese)](https://osu.ppy.sh/community/forums/topics/1260175)
-- [Livestream](https://live.bilibili.com/2996250 "Bilibili")
-- [Discord Server](https://discord.gg/uBHK5vq "Discord")
-- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1xgxKNKoJNEQz_zlPUf-4fyxeBDpuhfkgqxXyOxaIkpU "Google Sheets")**
+- [Livestream](https://live.bilibili.com/2996250)
+- [Discord Server](https://discord.gg/uBHK5vq)
+- [Challonge bracket](https://challonge.com/zh_CN/MCNC2021)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1xgxKNKoJNEQz_zlPUf-4fyxeBDpuhfkgqxXyOxaIkpU)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:61]:
```suggestion
- [Livestream](https://live.bilibili.com/2996250 "Bilibili")
- [Discord Server](https://discord.gg/uBHK5vq "Discord")
- [Challonge bracket](https://challonge.com/zh_CN/MCNC2021 "Challonge")
- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1xgxKNKoJNEQz_zlPUf-4fyxeBDpuhfkgqxXyOxaIkpU "Google Sheets")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 3]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_CN] [yuwenhao1](https://osu.ppy.sh/users/14318312) |
+
+![MCNC 2021 podium](img/podium.jpg)
+![MCNC 2021 bracket](img/bracket.jpg)
 
 ## Mappools
 
+### Finals
+
+**[Download the mappack here! (255 MB)](https://drive.google.com/file/d/1Fb0s3GTJzZ0v2LgCJLhtjphwpXXPYpkl/view?usp=sharing)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:126]:
```suggestion
**[Download the mappack here! (255 MB)](https://drive.google.com/file/d/1Fb0s3GTJzZ0v2LgCJLhtjphwpXXPYpkl/view?usp=sharing "Google Drive")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 4]
```diff
@@ -17,8 +18,8 @@ The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) is a country-ba
 | Registration phase | 2021-03-01/2021-03-31 |
 | Qualifiers | 2021-04-05/2021-04-11 |
 | Live drawings | 2021-04-11 (20:00 UTC+8) |
-| Group stage week 1 | 2021-04-12/2021-04-18 |
-| Group stage week 2 | 2021-04-19/2021-04-25 |
+| Group Stage week 1 | 2021-04-12/2021-04-18 |
+| Group Stage week 2 | 2021-04-19/2021-04-25 |
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:22]:
```suggestion
| Group stage week 1 | 2021-04-12/2021-04-18 |
| Group stage week 2 | 2021-04-19/2021-04-25 |
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 5]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [Hatsuki Yura - Hameln (Muses) \[Fanger\]](https://osu.ppy.sh/beatmapsets/1481462#mania/3038941)
+  - [Falcom Sound Team jdk - GENS D'ARMES (lemonguy) \[Gendarme\]](https://osu.ppy.sh/beatmapsets/1199166#mania/2497240)
+  - [Camellia - B L A C K - R A Y (Toaph Daddy) \[D A R K - M A G I C I A N x1.1\]](https://osu.ppy.sh/beatmapsets/1306124#mania/2707897)
+  - [Chroma - Lost found melody (Ciel) \[Found\]](https://osu.ppy.sh/beatmapsets/1051052#mania/2196635)
+- Tiebreaker
+  - [Camellia - NIGHTMARE CITY (\[Crz\]xz1z1z) \[NIGHTMARE\]](https://osu.ppy.sh/beatmapsets/1486342#mania/3047677)
+
+### Semifinals
+
+**[Download the mappack here! (160 MB)](https://drive.google.com/file/d/1tj0Q6dEO1YGqvsXZQ5huJkZAckPSh4fS/view?usp=sharing)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:150]:
```suggestion
**[Download the mappack here! (160 MB)](https://drive.google.com/file/d/1tj0Q6dEO1YGqvsXZQ5huJkZAckPSh4fS/view?usp=sharing "Google Drive")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 6]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [Shiraishi - True Millennium Queen (Muses) \[Scathach\]]( https://osu.ppy.sh/beatmapsets/1461646#mania/3002911)
+  - [Saitama Saisyu Heiki - L.L. Infinite Beating.ver (\[Illustrious\]) \[320% 1.1x\]](https://osu.ppy.sh/beatmapsets/1223208#mania/2562631)
+  - [Frums - dropdead (Chrubble) \[fatality\]](https://osu.ppy.sh/beatmapsets/957823#mania/2186287)
+  - [ONE – Future (arcwinolivirus) \[4K SV Innovation\]](https://osu.ppy.sh/beatmapsets/708785#mania/1498464)
+- Tiebreaker
+  - [Camellia vs. lapix - Solvicious (Mameyodoufu Remix) (Dubstek) \[Cybernetic Extinction\]](https://osu.ppy.sh/beatmapsets/933845#mania/1949498)
+
+### Quarterfinals
+
+**[Download the mappack here! (112 MB)](https://drive.google.com/file/d/12P98_uqoBN5qaqL6-4DSFtxyFwcrjtVo/view?usp=sharing)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:174]:
```suggestion
**[Download the mappack here! (112 MB)](https://drive.google.com/file/d/12P98_uqoBN5qaqL6-4DSFtxyFwcrjtVo/view?usp=sharing "Google Drive")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 7]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [WolF.R - Snow Wish (Muses) \[Frozen edit\]](https://osu.ppy.sh/beatmapsets/1451720#mania/2985094)
+  - [Sangatsu no Phantasia – Rendezvous (Muses) \[Jump! Peace! Shine! \[1.1x Rate\]\]](https://osu.ppy.sh/beatmapsets/1453455#mania/2988127)
+  - [SHK - Wedding Crashers (\[Crz\]xz1z1z) \[NTR\]](https://osu.ppy.sh/beatmapsets/1454389#mania/2989898)
+  - [sakuraburst – anticrystal (Mipha-) \[spooky\]](https://osu.ppy.sh/beatmapsets/1223208#mania/2562631)
+- Tiebreaker
+  - [rejection - Signal (feat. Such) (Mameyodoufu Remix) (\[OSC\]Amagai) \[ Keen Perception \[Revised Again\]\]](https://osu.ppy.sh/beatmapsets/1404380#mania/2993544)
+
+### Round of 16
+
+**[Download the mappack here! (123 MB)](https://drive.google.com/file/d/1VDpFnjGwhyzIp0SvqUu6XEizSJ9FfGiD/view?usp=sharing)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:196]:
```suggestion
**[Download the mappack here! (123 MB)](https://drive.google.com/file/d/1VDpFnjGwhyzIp0SvqUu6XEizSJ9FfGiD/view?usp=sharing "Google Drive")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 8]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [Erehamonika - Der Wald (\_underjoy) \[Verloren\]](https://osu.ppy.sh/beatmapsets/1319005#mania/2732397)
+  - [Hanatan - Palette (Lynessa) \[x1.0\]](https://osu.ppy.sh/beatmapsets/1413004#mania/2912596)
+  - [M2U - Felis (Nyakkeru) \[xz1z1z's Scurrying\]](https://osu.ppy.sh/beatmapsets/1321042#mania/2773340)
+  - [sakuzyo - Hodia (Tidek) \[Vivace\]](https://osu.ppy.sh/beatmapsets/924601#mania/1931227)
+- Tiebreaker
+  - [Kanone vs Yooh - Cold Inflaction (Extended Mix) (\[Crz\]xz1z1z) \[Cold But With You\]](https://osu.ppy.sh/beatmapsets/1274517#mania/2648242)
+
+### Group Stage
+
+**[Download the mappack here! (83 MB)](https://drive.google.com/file/d/13I8C26KAv3xVXBov1fRHuWR80Hskzi6T/view?usp=sharing)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:218]:
```suggestion
**[Download the mappack here! (83 MB)](https://drive.google.com/file/d/13I8C26KAv3xVXBov1fRHuWR80Hskzi6T/view?usp=sharing "Google Drive")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 9]
```diff
@@ -46,28 +49,298 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [nameless x toa - Patchwork Staccato (\[Crz\]Satori) \[ln\]](https://osu.ppy.sh/beatmapsets/1426977#mania/2938241)
+  - [succducc - me & u (Shoegazer) \[yume\]](https://osu.ppy.sh/beatmapsets/781107#mania/1640584)
+  - [Elijah Hill - Better (H1Pur) \[nearing perfection\]](https://osu.ppy.sh/beatmapsets/1376566#mania/2845109)
+- Tiebreaker
+  - [Camellia - Dance with Silence (\[Crz\]xz1z1z) \[Silence\]](https://osu.ppy.sh/beatmapsets/1405229#mania/2934744)
+
 ### Qualifiers
 
-**[Download the mappack here! (18 MB)](https://drive.google.com/file/d/1KYzdeiVoa11U0FYwjzqu-EKnDiA8DFr4/view?usp=sharing "Google Drive")**
+**[Download the mappack here! (18 MB)](https://drive.google.com/file/d/1KYzdeiVoa11U0FYwjzqu-EKnDiA8DFr4/view?usp=sharing)**
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:238]:
```suggestion
**[Download the mappack here! (18 MB)](https://drive.google.com/file/d/1KYzdeiVoa11U0FYwjzqu-EKnDiA8DFr4/view?usp=sharing "Google Drive")**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 10]
```diff
@@ -30,13 +31,15 @@ The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) is a country-ba
 
 | Placing | Prizes |
 | :-: | :-- |
-| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | CNY 2,500, unique profile badge, 4 months of osu!supporter |
-| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | CNY 1,500, 2 months of osu!supporter |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | CNY 800, 1 month of osu!supporter |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | CNY 2,500, unique profile badge, 4 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | CNY 1,500, 2 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | CNY 800, 1 month of osu!supporter tag |
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:36]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | CNY 2,500, unique profile badge, 4 months of osu!supporter |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | CNY 1,500, 2 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | CNY 800, 1 month of osu!supporter |
```

File: wiki/Tournaments/MCNC/2020/en.md [hunk 11]
```diff
@@ -34,6 +35,8 @@ The **osu!mania 4K Chinese National Cup 2020** (***MCNC 2020***) was a country-b
 | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | CNY 800, 10% off coupon for the New Crazyboard, 2 months of osu!supporter |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | CNY 400, 5% off coupon for the New Crazyboard, 1 months of osu!supporter |
 
+![](img/172-80.png "MCNC 2020 badge")
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2020/en.md:38]: also rename the file to `badge.png`

```suggestion
![](img/172-80.png "MCNC 2020 winner badge")
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 12]
```diff
@@ -30,13 +31,15 @@ The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) is a country-ba
...
+
+![MCNC 2021 winner badge](img/172-80.png)
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:38]: rename file to `badge.png`

File: wiki/Tournaments/MCNC/2021/en.md [hunk 13]
```diff
@@ -127,3 +400,6 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
 [flag_CN]: /wiki/shared/flag/CN.gif "China"
 [flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
 [flag_US]: /wiki/shared/flag/US.gif "United States"
+[flag_HK]: /wiki/shared/flag/HK.gif "Hong Kong"
+[flag_TW]: /wiki/shared/flag/TW.gif "Taiwan"
+[flag_MO]: /wiki/shared/flag/MO.gif "Macau"
```
Comment (Walavouchey) [inline wiki/Tournaments/MCNC/2021/en.md:405]: sorted

```suggestion
[flag_CN]: /wiki/shared/flag/CN.gif "China"
[flag_HK]: /wiki/shared/flag/HK.gif "Hong Kong"
[flag_MO]: /wiki/shared/flag/MO.gif "Macau"
[flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
[flag_TW]: /wiki/shared/flag/TW.gif "Taiwan"
[flag_US]: /wiki/shared/flag/US.gif "United States"
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
please note that the old [tournament template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/Template/en.md) isn't consistent with current articles and is soon to be [replaced](https://github.com/ppy/osu-wiki/pull/5747)

**Comment by Bazzadwarf** [comment]
The tournaments page table also needs updating to add the podiums for 2020 and 2021.

```
| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [osu!mania 4K Chinese National Cup 2019](MCNC/2019) | 2019-06-19 | 2019-08-12 | ![][flag_HK] [\[Crz\]Murasame](https://osu.ppy.sh/users/11563203) | ![][flag_TW] [\[Crz\]FolAH1217](https://osu.ppy.sh/users/6232458) | ![][flag_CN] [\[Crz\]Satori](https://osu.ppy.sh/users/7082178) |
| [osu!mania 4K Chinese National Cup 2020](MCNC/2020) | 2020-04-27 | 2020-06-28 | ![][flag_CN] [WLYMinato](https://osu.ppy.sh/users/12703319) | ![][flag_CN] [Kakuko](https://osu.ppy.sh/users/10413362) | ![][flag_CN] [\[Crz\]I-Night](https://osu.ppy.sh/users/8790619) |
| [osu!mania 4K Chinese National Cup 2021](MCNC/2021) | 2021-04-05 | 2021-06-06 | ![][flag_CN] [\[Crz\]sunnyxxy](https://osu.ppy.sh/users/10333739) | ![][flag_CN] [\[Crz\]Caicium](https://osu.ppy.sh/users/10702235) | ![][flag_CN] [yuwenhao1](https://osu.ppy.sh/users/14318312) |
``

File: wiki/Tournaments/MCNC/2021/en.md [hunk 14]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+| #25 | ![][flag_CN] [Littlesheep37](https://osu.ppy.sh/users/14742881) |
+| #26 | ![][flag_CN] [Sern888](https://osu.ppy.sh/users/2089244) |
+| #27 | ![][flag_MO] [\[Crz\]macaoleaf](https://osu.ppy.sh/users/7082716) | 
+| #28 | ![][flag_CN] [spensll](https://osu.ppy.sh/users/4547744) |
+| #29 | ![][flag_CN] [\[Paw\]Chuixing](https://osu.ppy.sh/users/14906054) |
+| #30 | ![][flag_TW] [Ha0201](https://osu.ppy.sh/users/12243730) |
+| #31 | ![][flag_TW] [Joylintp](https://osu.ppy.sh/users/13579078) |
+| #32 | ![][flag_CN] [4kGameBye](https://osu.ppy.sh/users/89545) |
+
+### Groups
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:100]:
```suggestion
## Groups
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 15]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_CN] [\[Crz\]sunnyxxy](https://osu.ppy.sh/users/10333739) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_CN] [\[Crz\]Caicium](https://osu.ppy.sh/users/10702235) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_CN] [yuwenhao1](https://osu.ppy.sh/users/14318312) |
+
+![MCNC 2021 podium](img/podium.jpg)
+![MCNC 2021 bracket](img/bracket.jpg)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:120]:
```suggestion
![](img/bracket.jpg "MCNC 2021 bracket")

## Podium

![](img/podium.jpg "MCNC 2021 podium")
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 16]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
 ## Mappools
 
+### Finals
+
+**[Download the mappack here! (255 MB)](https://drive.google.com/file/d/1Fb0s3GTJzZ0v2LgCJLhtjphwpXXPYpkl/view?usp=sharing "Google Drive")**
+
+- FreeMod
+  - [Camellia - Racemization (Ayase Eli) \[ELPIS' Racemization 1.15x\]](https://osu.ppy.sh/beatmapsets/1484517#mania/3044452)
+  - [sakuzyo - PANDORA PARADOXXX (Imperial Wolf) \[4K Re:MASTER LV.15\]](https://osu.ppy.sh/beatmapsets/1483812#mania/3043161)
+  - [Camellia - Flamewall (Cut ver.) (\[Crz\]xz1z1z) \[AlexDunk's FINAL REBELLION (Cut Edit) \[1.05x Rate\]](https://osu.ppy.sh/beatmapsets/1462775#mania/3019256)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:131]:
```suggestion
  - [Camellia - Flamewall (Cut ver.) (\[Crz\]xz1z1z) \[AlexDunk's FINAL REBELLION (Cut Edit) \[1.05x Rate\]\]](https://osu.ppy.sh/beatmapsets/1462775#mania/3019256)
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 17]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [konomi suzuki - CHOIR JAIL (Muses) \[Sin edit\]](https://osu.ppy.sh/beatmapsets/1392626#mania/3038977)
+  - [Boys Like Girls - The Great Escape (Shoegazer) \[Departure 1.2x\]](https://osu.ppy.sh/beatmapsets/1186973#mania/2476815)
+  - [Camellia - Kamah (Scythe) (Ticle) \[Set them free\]](https://osu.ppy.sh/beatmapsets/1314013#mania/2723400)
+  - [LV.4 - Angel Dust (2008 Mix) (\[Crz\]xz1z1z) \[Cut Edit\]](https://osu.ppy.sh/beatmapsets/1382971#mania/3019151)
+  - [Hatsuki Yura - Hameln (Muses) \[Fanger\]](https://osu.ppy.sh/beatmapsets/1481462#mania/3038941)
+  - [Falcom Sound Team jdk - GENS D'ARMES (lemonguy) \[Gendarme\]](https://osu.ppy.sh/beatmapsets/1199166#mania/2497240)
+  - [Camellia - B L A C K - R A Y (Toaph Daddy) \[D A R K - M A G I C I A N x1.1\]](https://osu.ppy.sh/beatmapsets/1306124#mania/2707897)
+  - [Chroma - Lost found melody (Ciel) \[Found\]](https://osu.ppy.sh/beatmapsets/1051052#mania/2196635)
+- Tiebreaker
+  - [Camellia - NIGHTMARE CITY (\[Crz\]xz1z1z) \[NIGHTMARE\]](https://osu.ppy.sh/beatmapsets/1486342#mania/3047677)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:146]:
```suggestion
  - **[Camellia - NIGHTMARE CITY (\[Crz\]xz1z1z) \[NIGHTMARE\]](https://osu.ppy.sh/beatmapsets/1486342#mania/3047677)**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 18]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [YOASOBI - Ano Yume o Nazotte (Muses) \[Fall into the Night\]](https://osu.ppy.sh/beatmapsets/1135500#mania/2474910)
+  - [Train - 50 Ways to Say Goodbye (\[Crz\]xz1z1z) \[Muses was my everything\]]( https://osu.ppy.sh/beatmapsets/1459444#mania/2998939)
+  - [Feryquitous - Arcahv (Abraxos) \[Rend\]]( https://osu.ppy.sh/beatmapsets/1184106#mania/2468487)
+  - [Aoi Eir - Cynthia No Hikari (Ayase Eli) \[arpia97's Illumination 4K 1.21x\]](https://osu.ppy.sh/beatmapsets/1462729#mania/3004812)
+  - [Shiraishi - True Millennium Queen (Muses) \[Scathach\]]( https://osu.ppy.sh/beatmapsets/1461646#mania/3002911)
+  - [Saitama Saisyu Heiki - L.L. Infinite Beating.ver (\[Illustrious\]) \[320% 1.1x\]](https://osu.ppy.sh/beatmapsets/1223208#mania/2562631)
+  - [Frums - dropdead (Chrubble) \[fatality\]](https://osu.ppy.sh/beatmapsets/957823#mania/2186287)
+  - [ONE – Future (arcwinolivirus) \[4K SV Innovation\]](https://osu.ppy.sh/beatmapsets/708785#mania/1498464)
+- Tiebreaker
+  - [Camellia vs. lapix - Solvicious (Mameyodoufu Remix) (Dubstek) \[Cybernetic Extinction\]](https://osu.ppy.sh/beatmapsets/933845#mania/1949498)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:170]:
```suggestion
  - **[Camellia vs. lapix - Solvicious (Mameyodoufu Remix) (Dubstek) \[Cybernetic Extinction\]](https://osu.ppy.sh/beatmapsets/933845#mania/1949498)**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 19]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+
+### Quarterfinals
+
+**[Download the mappack here! (112 MB)](https://drive.google.com/file/d/12P98_uqoBN5qaqL6-4DSFtxyFwcrjtVo/view?usp=sharing "Google Drive")**
+
+- FreeMod
+  - [Dimrain47 – Infernoplex (\[Crz\]xz1z1z) \[Mat's Insane 1.35x\]](https://osu.ppy.sh/beatmapsets/https://osu.ppy.sh/beatmapsets/1454397#mania/2989914)
+  - [Camellia - Alone intelligence (guden) \[Solace\]](https://osu.ppy.sh/beatmapsets/1235533#mania/2568385)
+  - [Junk - Life is PIANO (Muses & xz's Life is JACKKK) \[\[Crz\]xz1z1z\]](https://osu.ppy.sh/beatmapsets/1369788#mania/2832368)
+  - [Hino Isuka - #be_fortunate (HowToPlayLN) \[#be_edited\]](https://osu.ppy.sh/beatmapsets/1099229#mania/2349916)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:180]:
```suggestion
  - [Hino Isuka - #be\_fortunate (HowToPlayLN) \[#be\_edited\]](https://osu.ppy.sh/beatmapsets/1099229#mania/2349916)
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 20]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [sakuraburst – dragonlands (Toaph Daddy) \[myth\]](https://osu.ppy.sh/beatmapsets/808881#mania/1697375)
+  - [Morimori – PUPA (Fullerene-) \[Metamorphosis\]](https://osu.ppy.sh/beatmapsets/492056#mania/1048321)
+  - [Yooh – Heracles (Sherie) \[Confusion\]](https://osu.ppy.sh/beatmapsets/1135500#mania/2474910)
+  - [Honda Michiyo - close your eyes (\[Crz\]xz1z1z) \[Charoi's Beginner 1.1x\]](https://osu.ppy.sh/beatmapsets/1454360#mania/2989810)
+  - [WolF.R - Snow Wish (Muses) \[Frozen edit\]](https://osu.ppy.sh/beatmapsets/1451720#mania/2985094)
+  - [Sangatsu no Phantasia – Rendezvous (Muses) \[Jump! Peace! Shine! \[1.1x Rate\]\]](https://osu.ppy.sh/beatmapsets/1453455#mania/2988127)
+  - [SHK - Wedding Crashers (\[Crz\]xz1z1z) \[NTR\]](https://osu.ppy.sh/beatmapsets/1454389#mania/2989898)
+  - [sakuraburst – anticrystal (Mipha-) \[spooky\]](https://osu.ppy.sh/beatmapsets/1223208#mania/2562631)
+- Tiebreaker
+  - [rejection - Signal (feat. Such) (Mameyodoufu Remix) (\[OSC\]Amagai) \[ Keen Perception \[Revised Again\]\]](https://osu.ppy.sh/beatmapsets/1404380#mania/2993544)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:192]:
```suggestion
  - **[rejection - Signal (feat. Such) (Mameyodoufu Remix) (\[OSC\]Amagai) \[ Keen Perception \[Revised Again\]\]](https://osu.ppy.sh/beatmapsets/1404380#mania/2993544)**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 21]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [C-show - GIMME DA BLOOD (H1Pur) \[GIMME DA SV\]](https://osu.ppy.sh/beatmapsets/1428709#mania/2941393)
+  - [Fractal Dreamers - Gardens Under a Spring Sky (\[Crz\]xz1z1z) \[Vernal\]](https://osu.ppy.sh/beatmapsets/1398608#mania/2885730)
+  - [YOASOBI - Yoru ni Kakeru (11Bit) \[a longly night... 1.1x\]](https://osu.ppy.sh/beatmapsets/1211012#mania/2573485)
+  - [Mian - Present (Doshowz) \[Hard\]](https://osu.ppy.sh/beatmapsets/1444039#mania/2970437)
+  - [Erehamonika - Der Wald (\_underjoy) \[Verloren\]](https://osu.ppy.sh/beatmapsets/1319005#mania/2732397)
+  - [Hanatan - Palette (Lynessa) \[x1.0\]](https://osu.ppy.sh/beatmapsets/1413004#mania/2912596)
+  - [M2U - Felis (Nyakkeru) \[xz1z1z's Scurrying\]](https://osu.ppy.sh/beatmapsets/1321042#mania/2773340)
+  - [sakuzyo - Hodia (Tidek) \[Vivace\]](https://osu.ppy.sh/beatmapsets/924601#mania/1931227)
+- Tiebreaker
+  - [Kanone vs Yooh - Cold Inflaction (Extended Mix) (\[Crz\]xz1z1z) \[Cold But With You\]](https://osu.ppy.sh/beatmapsets/1274517#mania/2648242)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:214]:
```suggestion
  - **[Kanone vs Yooh - Cold Inflaction (Extended Mix) (\[Crz\]xz1z1z) \[Cold But With You\]](https://osu.ppy.sh/beatmapsets/1274517#mania/2648242)**
```

File: wiki/Tournaments/MCNC/2021/en.md [hunk 22]
```diff
@@ -46,18 +49,190 @@ The osu!mania 4K Chinese National Cup 2021 is run by various community members.
...
+  - [iconoclasm - perditus paradisus (lemonguy) \[Another\]](https://osu.ppy.sh/beatmapsets/1302861#mania/2845681)
+  - [Helblinde - Above The Clouds (\[Crz\]xz1z1z) \[Tidek's Heaven (Edit)\]](https://osu.ppy.sh/beatmapsets/1427003#mania/2938278)
+  - [xi - Fiat Lux (\[Crz\]MemoryI) \[on the same horizon\]](https://osu.ppy.sh/beatmapsets/1425149#mania/2934737)
+  - [Sta - The Silence (Wh1teh) \[Tranquility\]](https://osu.ppy.sh/beatmapsets/451169#mania/967884)
+  - [Unlucky Morpheus - Danzai wa Amaneku Ningen no Motto ni (\[Illustrious\]) \[kami no breeze\]](https://osu.ppy.sh/beatmapsets/1018449#mania/2902249)
+  - [nameless x toa - Patchwork Staccato (\[Crz\]Satori) \[ln\]](https://osu.ppy.sh/beatmapsets/1426977#mania/2938241)
+  - [succducc - me & u (Shoegazer) \[yume\]](https://osu.ppy.sh/beatmapsets/781107#mania/1640584)
+  - [Elijah Hill - Better (H1Pur) \[nearing perfection\]](https://osu.ppy.sh/beatmapsets/1376566#mania/2845109)
+- Tiebreaker
+  - [Camellia - Dance with Silence (\[Crz\]xz1z1z) \[Silence\]](https://osu.ppy.sh/beatmapsets/1405229#mania/2934744)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2021/en.md:234]:
```suggestion
  - **[Camellia - Dance with Silence (\[Crz\]xz1z1z) \[Silence\]](https://osu.ppy.sh/beatmapsets/1405229#mania/2934744)**
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**